nomad/provider.go

```diff
@@ -52,6 +68,36 @@ func Provider() terraform.ResourceProvider {
 	}
 }
 
+// populateTokenPath figures out the token path using homedir to get the user's
+// home directory
```

Comment:
Is this a file/pattern taken from elsewhere? From what I can tell, Vault doesn't do this in the CLI. I'd like to try to match as closely as possible what Vault does, I think, if for no other reason than it doesn't encourage developers to store sensitive data on their hard drives/in their CI systems.